quandl/get_table.py

```diff
@@ -6,7 +6,7 @@
 import os
 
 
-def get_table(code, **options):
+def get_table(datatable_code, **options):
```

Comment:
My python must be rusty. Why does changing the variable from `code` to `datatable_code` fix this issue?